keen/api.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
You got rid of the `print()` calls at some point, I believe.

Reply:
thanks, will fix.